### ISSUE TITLE
[homebridge-alexa-smarthome] After initialization - HttpError(Error getting smart home device state. Reason: Timeout)

### ISSUE THREAD

**Issue by dhruvarora6670** [joeyhage/homebridge-alexa-smarthome#138]
[homebridge-alexa-smarthome] After initialization - HttpError(Error getting smart home device state. Reason: Timeout)

**Logs:**

```
[homebridge-alexa-smarthome] After initialization - HttpError(Error getting smart home device state. Reason: Timeout)
```

**Plugin Config:**

```json
{
    "devices": [],
    "excludeDevices": [],
    "auth": {
        "refreshInterval": 4,
        "proxy": {
            "clientHost": "localhost",
            "port": 9000
        }
    },
    "amazonDomain": "amazon.in",
    "language": "en-US",
    "performance": {
        "cacheTTL": 30,
        "backgroundRefresh": false
    },
    "debug": false,
    "platform": "HomebridgeAlexaSmartHome",
    "_bridge": {
        "username": "0E:7E:16:C6:9A:87",
        "port": 57584
    }
}
```

**Comment by joeyhage** [comment]
For those using 2.2.0-alpha.0, I plan to release 2.2.0-alpha.1 in a few hours. This will add support for all the devices previously supported and should address many of the issues with the first alpha. Thank you for your patience and support

**Comment by mynameisbrendan** [comment]
I just upgraded to v2.2.0-alpha.2 and I still receive the error below for my smartlock:

`[7/8/2024, 3:14:55 PM] [HomebridgeAlexaSmartHome] Initialize Devices - UnsupportedDeviceError(Unsupported device: Front Door with type: SMARTLOCK. Device type is either unsupported or device does not support all the required operations. Device supported operations: lock,unlock. Currently supported device types are: LIGHT,SWITCH,SMARTLOCK,FAN,SMARTPLUG,THERMOSTAT,ALEXA_VOICE_ENABLED,AIR_QUALITY_MONITOR,VACUUM_CLEANER,GAME_CONSOLE,AIR_FRESHENER.)`

**Comment by Jager422** [comment]
I’m receiving the same error for my thermostat.

[7/7/2024, 11:11:22 PM] [homebridge-alexa-smarthome] Initialize Devices - UnsupportedDeviceError(Unsupported device: Thermostat with type: THERMOSTAT. Device type is either unsupported or device does not support all the required operations. Device supported operations: setTargetSetpoint,adjustTargetSetpoint,setThermostatMode. Currently supported device types are: LIGHT,SWITCH,SMARTLOCK,FAN,SMARTPLUG,THERMOSTAT,ALEXA_VOICE_ENABLED,AIR_QUALITY_MONITOR,VACUUM_CLEANER,GAME_CONSOLE,AIR_FRESHENER.)

**Comment by fmottaz** [comment]
v2.2.0-alpha.2 is doing fine here. A slight delay when first opening Home, but otherwise a log which remains pleasantly empty after startup. Thanks for the work.

**Comment by joeyhage** [comment]
> I just upgraded to v2.2.0-alpha.2 and I still receive the error below for my smartlock:

@mynameisbrendan I think I've fixed that issue in 2.2.0-alpha.4

**Comment by joeyhage** [comment]
> I’m receiving the same error for my thermostat.

This should be fixed in either 2.2.0-alpha.3 or 2.2.0-alpha.4

**Comment by Jager422** [comment]
@joeyhage it appears to be working now
